source/_components/azure_event_hub.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Azure Event Hub"
+description: "Setup for Azure Event Hub integration"
+date: 2019-05-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: azure_event_hub.svg
+ha_category:
+  - History
+ha_release: 0.93
```

Comment:
```suggestion
ha_release: 0.94
```

Reply:
Why is this, is .93 fixed already?

Reply:
The cutoff for the 0.93 release was already 2 weeks ago 😉